### PR TITLE
add system env for trace ignore plugin

### PR DESCRIPTION
**Pull request by a198720**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [* ] New feature provided
- [x] Improve performance

- Related issues

___
### Bug fix
- Bug description.
add system env for trace ignore plugin
- How to fix?
add system env for trace ignore plugin
___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java [hunk 1]
```diff
@@ -57,6 +60,18 @@ public static void initialize() throws ConfigNotFoundException, AgentPackageNotF
             configFileStream = loadConfigFromAgentFolder();
             Properties properties = new Properties();
             properties.load(configFileStream);
+            PropertyPlaceholderHelper helper =
```
Comment (wu-sheng) [inline apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java]: @JaredTan95 Is this right usage of your util API? Could you help on review this PR?

Comment (JaredTan95) [reply apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java]: The usage is right. BTW, @a198720 you can add some test case in `org.apache.skywalking.apm.plugin.trace.ignore.TraceIgnoreTest`? Take `org.apache.skywalking.apm.agent.core.conf.SnifferConfigInitializerTest#testConfigOverridingFromSystemEnv` as a example. :D

Comment (wu-sheng) [reply apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java]: OK. If you are going to add test. I will wait until you done. Thanks for the contributions.

File: apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java [hunk 2]
```diff
@@ -57,6 +60,18 @@ public static void initialize() throws ConfigNotFoundException, AgentPackageNotF
...
+                    new PropertyPlaceholderHelper(PlaceholderConfigurerSupport.DEFAULT_PLACEHOLDER_PREFIX,
```
[the remark this answers is not in the document]
Comment (JaredTan95) [reply apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java]: ok, Soon or later.

**Comment by JaredTan95** [comment]
@a198720 FYI, I refactored a little about  `PropertyPlaceholderHelper ` class.

**Comment by wu-sheng** [comment]
@a198720 Sorry. Need you to refactor your codes to fit new API. I forgot to merge this first.

**Comment by wu-sheng** [comment]
@a198720 Do you want to continue this contribution? @JaredTan95 How about starting to new PR to fix this?

**Pull request merged**